.github/workflows/generate-changelog.yml

```diff
@@ -25,8 +26,6 @@ jobs:
             else
               MSG="Update CHANGELOG.md for #${{ github.event.pull_request.number }}"
             fi
-            git config --local user.email changelogbot@mongodb.com
-            git config --local user.name changelogbot
```

Comment:
does this have to be removed? Not sure if the svc account details would be used automatically, we can try.

Reply:
As per my understanding, setting the token in the checkout action is enough to be authenticated with that account.
description of token attribute of the checkout action _Personal access token (PAT) used to fetch the repository. The PAT is configured with the local git config, which enables your scripts to run authenticated git commands. The post-job step removes the PAT._

Reply:
Correct, my doubt is if git will still be missing author information for the actual commit. Either way we can try and adjust if needed.